Packs/Microsoft_SharePoint_ToolShell_Vulnerability/Playbooks/playbook-Cortex_XDR_-_CVE-2025-49704_and_CVE-2025-49706_and_CVE-2025-53770_and_CVE-2025-53771_-_Microsoft_SharePoint_ToolShell_vulnerability_chain.yml

```diff
@@ -0,0 +1,3529 @@
+    id: "138"
+    taskid: 985dc562-1311-4f8a-8df7-f06d6c2cdaca
+    type: playbook
+    task:
+      id: 985dc562-1311-4f8a-8df7-f06d6c2cdaca
+      version: -1
+      name: Panorama Query Logs for Related Session
+      description: 'Query Panorama Logs of types: traffic, threat, url, data-filtering and wildfire.'
+      playbookName: Panorama Query Logs
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "28"
+    scriptarguments:
+      log_type:
+        simple: threat
+      query:
+        simple: '(threatid eq 96436) or (threatid eq 96481) '
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1012.5,
+          "y": 3985
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "139":
+    id: "139"
+    taskid: 87c1c9c8-084b-4197-89b8-f7612b4c4aca
+    type: regular
+    task:
+      id: 87c1c9c8-084b-4197-89b8-f7612b4c4aca
+      version: -1
+      name: Set Potential Agent IDs to search on
+      description: "Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.\n\nThis automation runs using the default Limited User role, unless you explicitly change the permissions.\nFor more information, see the section about permissions here:\n- For Cortex XSOAR 6 see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations \n- For Cortex XSOAR 8 Cloud see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script\n- For Cortex XSOAR 8.7 On-prem see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script"
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "135"
+    scriptarguments:
+      key:
+        simple: agentsidstocheck2
+      value:
+        complex:
+          root: alert
+          accessor: agentid
+          transformers:
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: incident.agentsid
+                iscontext: true
+          - operator: RemoveEmpty
+            args:
+              empty_values: {}
+              remove_keys:
+                value:
+                  simple: "true"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4092.5,
+          "y": 2775
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "140":
```

Comment:
please change the arguments values to ExtractedIndicators
<img width="464" height="1059" alt="image" src="https://github.com/user-attachments/assets/00e6fe09-0666-49b0-8526-250c04e9f48f" />